src/kafka-util/src/bin/kgen.rs

```diff
@@ -102,12 +125,26 @@ fn main() -> anyhow::Result<()> {
     let dd = Uniform::new_inclusive(0, 255);
     let mut buf = vec![];
     let mut rng = thread_rng();
+
+    let keys_random = matches.value_of("keys").unwrap() == "random";
+    let key_dist = if keys_random {
+        let key_min: u64 = matches.value_of("key-min").unwrap().parse()?;
+        let key_max: u64 = matches.value_of("key-max").unwrap().parse()?;
+        Some(Uniform::new_inclusive(key_min, key_max))
```

Comment:
will this handle checking that key_min < key_max

Reply:
It will panic with "Uniform::new called with `low >= high`" , which I think is fine behavior for now.